string description
-----
string object_name
geometry_msgs/Pose pose
geometry_msgs/Twist twist
string explanation
bool success